onnxruntime/core/providers/openvino/backends/vadm_backend.cc

```diff
@@ -62,7 +62,7 @@ VADMBackend::VADMBackend(const ONNX_NAMESPACE::ModelProto& model_proto,
   }
 #endif
 
-#if defined(OPENVINO_2020_4)
+#if defined(OPENVINO_2021_1) || defined(OPENVINO_2021_2) || defined(OPENVINO_2021_3) || defined(OPENVINO_2021_4)
   if (const_outputs_map_.size() == subgraph_context_.output_names.size())
```

Comment:
is this correct?
previously this if block was only there for 2020.4 ?
and now it's there for newer than 2020.4 ?

Reply:
Yes George, This is correct. The check had to be there for all the currented supported versions of OpenVINO except OV 2020.3 version.